gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -466,15 +466,20 @@ gst_tensor_filter_common_set_property (GstTensorFilterPrivate * priv,
 
       /* 'custom-easy' framework has a virtual model file */
       if (g_strcmp0 (prop->fwname, "custom-easy") != 0) {
-        for (idx = 0; idx < prop->num_models; idx++) {
-          if (!g_file_test (prop->model_files[idx], G_FILE_TEST_IS_REGULAR)) {
-            g_critical ("Cannot find the model file [%d]: %s\n",
-                idx, prop->model_files[idx]);
+        if (priv->fw->check_model_path_in_plugin != 0) {
+          /* In this case, the general framework should check the validities of the path of model files */
+          for (idx = 0; idx < prop->num_models; idx++) {
+            if (!g_file_test (prop->model_files[idx], G_FILE_TEST_IS_REGULAR)) {
+              g_critical ("Cannot find the model file [%d]: %s\n",
+                  idx, prop->model_files[idx]);
+              return FALSE;
```

Comment:
I'm not sure it's okay to return a value here. IMO, basically, `set_propery` does not has a return value, but in case of `common_set_property`, its return value is used to check `prop_id`'s validity. So, I think this function's doxygen comment (related to return) and line 298 in tensor_filter.c also need to be changed, to support this feature.